src/components/MultiHopTrade/hooks/useReceiveAddress.tsx

```diff
@@ -52,7 +52,7 @@ export const useReceiveAddress = ({
     return false
   }, [buyAsset, wallet])
 
-  const { data: walletReceiveAddress, isLoading } = useQuery({
+  const { data: walletReceiveAddress, isLoading } = useQuery<string | null>({
```

Comment:
I was going to comment that implicit typing is not required, but I see the intent now 👀 very very smort ser!

Reply:
Yes indeed 😉